packages/plugins/plugin-automation/src/components/AutomationPanel/AutomationPanel.tsx

```diff
@@ -79,64 +78,60 @@ export const AutomationPanel = ({ space, object, initialTrigger, onDone }: Autom
   };
 
   return (
-    <Clipboard.Provider>
-      <StackItem.Content classNames='block overflow-y-auto'>
-        <div className='flex flex-col w-full'>
-          {trigger ? (
-            <ControlItem title={t('trigger editor title')}>
-              <TriggerEditor space={space} trigger={trigger} onSave={handleSave} onCancel={handleCancel} />
-            </ControlItem>
-          ) : (
-            <div role='none' className={controlItemClasses}>
-              <List.Root<FunctionTrigger> items={triggers} isItem={S.is(FunctionTrigger)} getId={(field) => field.id}>
-                {({ items: triggers }) => (
-                  <div role='list' className='flex flex-col w-full'>
-                    {triggers?.map((trigger) => {
-                      const copyAction = getCopyAction(client, trigger);
-                      return (
-                        <List.Item<FunctionTrigger>
-                          key={trigger.id}
-                          item={trigger}
-                          classNames={mx(grid, ghostHover, 'items-center', 'px-2')}
+    <div className='flex flex-col w-full'>
+      {trigger ? (
+        <ControlItem title={t('trigger editor title')}>
+          <TriggerEditor space={space} trigger={trigger} onSave={handleSave} onCancel={handleCancel} />
+        </ControlItem>
+      ) : (
+        <div role='none' className={controlItemClasses}>
+          <List.Root<FunctionTrigger> items={triggers} isItem={S.is(FunctionTrigger)} getId={(field) => field.id}>
+            {({ items: triggers }) => (
+              <div role='list' className='flex flex-col w-full'>
+                {triggers?.map((trigger) => {
+                  const copyAction = getCopyAction(client, trigger);
+                  return (
+                    <List.Item<FunctionTrigger>
+                      key={trigger.id}
+                      item={trigger}
+                      classNames={mx(grid, ghostHover, 'items-center', 'px-2')}
+                    >
+                      <Input.Root>
+                        <Input.Switch
+                          checked={trigger.enabled}
+                          onCheckedChange={(checked) => (trigger.enabled = checked)}
+                        />
+                      </Input.Root>
+
+                      <div className={'flex'}>
```

Comment:
More tech debt probably, but it’s okay to put buttons inside headings & titles, so this interceding div should not be necessary.

Reply:
same here